packages/presentation/src/components/Avatar.svelte

```diff
@@ -17,15 +17,17 @@
   import chen from '../../img/chen.png'
 
   export let avatar: any = chen
-  export let size: 16 | 24 | 32 | 34 | 36 | 88 = 24
+  export let size: 16 | 24 | 28 | 32 | 34 | 36 | 88 = 24
+
+  let remSize = size / parseFloat(getComputedStyle(document.documentElement).fontSize)
```

Comment:
Please remove everything related to `px` -- avatar sizes should be `small`, etc. Ideally these sizes should match icon sizes.